coordinates/sites.json

```diff
@@ -872,5 +872,17 @@
         "longitude": 10.504496611198473,
         "longitude_unit": "degree",
         "source": "LALSuite: https://git.ligo.org/lscsoft/lalsuite/-/blob/lalsuite-v6.70/lal/lib/tools/LALDetectors.h"
+    },
+    "ovro": {
+        "name": "Owens Valley Radio Observatory",
+        "aliases": ["mma"],
+        "timezone": "US/Pacific",
+        "elevation": 1188.6,
```

Comment:
I notice that this differs slightly from the numbers on the public website: https://www.ovro.caltech.edu/
- but only ever so slightly. I think "OVRO staff measurement" is a good source, I just wanted to make sure that that difference is known, and the number here is actually the better number to use.

Reply:
Yes, this differs slightly from the web site, but we'd like to update it. I am an OVRO staff member and we have discussed this internally.
Thanks for your help.